🗒️ [#5283] Rename form list-view

VinLe4 opened the pull request:
Closes #5283 

[skip: e2e]

**Changes**

Renamed and removed columns from forms list-view as follows:
Active and Maintenance mode columns removed - replaced with Live
Translation enabled column removed
Maximum allowed submissions column removed
Authentication backend(s) renamed to inloggen
Payment backend removed
Registration backend(s) renamed to registrations

**Checklist**

Check off the items that are completed or not relevant.

- Impact on features

  - [x] Checked copying a form
  - [x] Checked import/export of a form
  - [x] Config checks in the configuration overview admin page
  - [x] Problem detection in the admin email digest is handled

- Release management

  - [x] I have labelled the PR as "needs-backport" accordingly

- I have updated the translations assets (you do NOT need to provide translations)

  - [x] Ran `./bin/makemessages_js.sh`
  - [x] Ran `./bin/compilemessages_js.sh`

- Dockerfile/scripts

  - [x] Updated the Dockerfile with the necessary scripts from the `./bin` folder

- Commit hygiene

  - [x] Commit messages refer to the relevant Github issue
  - [x] Commit messages explain the "why" of change, not the how

sergei-maertens commented on src/openforms/forms/admin/form.py:
```diff
@@ -122,6 +122,24 @@ def expected_parameters(self):
         return [self.parameter_name]
 
 
+class IsLiveListFilter(admin.SimpleListFilter):
```
This filter is not necessary - the existing filters were okay.

sergei-maertens commented on src/openforms/forms/admin/form.py:
```diff
@@ -270,6 +284,12 @@ def get_object_actions(self, obj: Form) -> str:
     def anno_name(self, obj: Form) -> str:
         return obj.admin_name
 
+    @admin.display(description=_("Live"))
+    def is_live(self, obj: Form) -> bool:
+        return obj.active and not obj._is_deleted
+
+    is_live.boolean = True
```
```suggestion
    @admin.display(description=_("Live"), boolean=True)
    def is_live(self, obj: Form) -> bool:
        return obj.active and not obj._is_deleted
```

sergei-maertens commented on src/openforms/forms/models/form.py:
```diff
@@ -467,9 +467,7 @@ def get_authentication_backends_display(self):
-    get_authentication_backends_display.short_description = _(
-        "authentication backend(s)"
-    )
+    get_authentication_backends_display.short_description = _("inloggen")
```
please use the English string, we'll manage translations near release :smiley:

VinLe4 commented:
Hi @sergei-maertens , I've made the change requests. I've removed the "IsLiveListFilter" and reintroduced the previous "active" filter. I've cleaned up the "live" filter logic in accordance to your suggestion. "inloggen" is now "logins".

sergei-maertens reviewed (approved):
Lovely, thank you for your contribution!

Feel free to add yourself to https://github.com/open-formulieren/open-forms/blob/master/AUTHORS as well!

sergei-maertens commented:
@VinLe4 can I mark this PR ready for review so we can merge it, or are you still doing additional things?

VinLe4 commented:
Hi @sergei-maertens, yes this can be marked ready for review! No more changes to be made.
